src/frontend/js/components/Modal/index.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
```

Comment:
Just a nitpick. 
I think it could be interesting to add control for `shouldCloseOnOverlayClick` and `shouldCloseOnEsc` props as this is something interesting to know when we are using modal.